config/locales/reference.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  reference_status:
+    not_requested_yet: 'NOT REQUESTED YET'
+    feedback_requested: 'REQUESTED'
+    feedback_provided: 'FEEDBACK PROVIDED'
+    feedback_refused: 'DECLINED'
```

Comment:
Because we already uppercase using CSS, these should all be title cased, which is better for screen readers in some niche cases:

```yml
      not_requested_yet: 'Not requested yet'
      feedback_requested: 'Requested'
      feedback_provided: 'Feedback provided'
      feedback_refused: 'Declined'
```